src/docdb/commands.ts

```diff
@@ -43,21 +43,23 @@ export class DocDBCommands {
         const masterKey = coll.db.getPrimaryMasterKey();
         const endpoint = coll.db.getEndpoint();
         const client = new DocumentClient(endpoint, { masterKey: masterKey });
-        const docid = await vscode.window.showInputBox({
+        const docID = await vscode.window.showInputBox({
```

Comment:
Use validateInput to check for empty string and specifically say something like "Empty string is not allowed".